MapboxDirections/MBAttribute.h

```diff
@@ -0,0 +1,17 @@
+/**
+ Attributes are metadata information for a given route. The number of attributes returned will be a direct 1-1 relationship with the route's full geometry. Each type will return an ordered list of requested attributes. For `.distance`, `.expectedTrabelTime`, and `.speed` there will be one less value when compared to the route geometry. This is because these values represent the data on segment between geometry points.
+*/
+typedef NS_OPTIONS(NSUInteger, MBAttributeOptions) {
+
+    /// Segment distance.
+    MBAttributeDistance = (1 << 1),
+    
+    // Segment expected travel time in seconds.
+    MBAttributeExpectedTravelTime = (1 << 2),
+    
+    // Segment current speed.
+    MBAttributeSpeed = (1 << 3),
+    
+    // Unique OpenStreetMap node.
```

Comment:
> Also note that this attribute option is added automatically when any of the other attributes are added.

This is not true: 
![image](https://cloud.githubusercontent.com/assets/1058624/24875269/69fe5244-1ddc-11e7-8202-dad69e28e662.png)